src/SFA.DAS.ProviderApprenticeshipsService.Web/DependencyResolution/CommitmentsRegistry.cs

```diff
@@ -17,16 +19,20 @@ public class CommitmentsRegistry : Registry
         public CommitmentsRegistry()
         {
             For<CommitmentsApiClientConfiguration>().Use(c => c.GetInstance<IAutoConfigurationService>().Get<CommitmentsApiClientConfiguration>(ConfigurationKeys.CommitmentsApiClient)).Singleton();
-            For<ICommitmentsApiClientConfiguration>().Use(c => c.GetInstance<CommitmentsApiClientConfiguration>());
-
-            For<ITrainingProgrammeApi>().Use<TrainingProgrammeApi>()
+            For<ICommitmentsApiClientConfiguration>().Use(c => c.GetInstance<CommitmentsApiClientConfiguration>());For<ITrainingProgrammeApi>().Use<TrainingProgrammeApi>()
                 .Ctor<HttpClient>().Is(c => GetHttpClient(c));
             
             For<IProviderCommitmentsApi>().Use<ProviderCommitmentsApi>()
               .Ctor<HttpClient>().Is(c => GetHttpClient(c));
 
             For<IValidationApi>().Use<ValidationApi>()
                 .Ctor<HttpClient>().Is(c => GetHttpClient(c));
+
+            For<PasForCommitmentsV2Configuration>().Use(c => c.GetInstance<IAutoConfigurationService>().Get<PasForCommitmentsV2Configuration>("SFA.DAS.ProviderApprenticeshipsService")).Singleton();
```

Comment:
please move the "SFA.DAS.ProviderApprenticeshipsService" move to ConfigurationKeys like for CommitmentsApiClient on line 21.